homeassistant/components/light/c_by_ge.py

```diff
@@ -0,0 +1,129 @@
+"""
+Support for C by GE lights.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.c_by_ge/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_NAME, CONF_PASSWORD
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, ATTR_COLOR_TEMP, SUPPORT_COLOR_TEMP,
+    Light, PLATFORM_SCHEMA)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util.color import \
+    color_temperature_kelvin_to_mired as kelvin_to_mired
+from homeassistant.util.color import \
+    color_temperature_mired_to_kelvin as mired_to_kelvin
+
+REQUIREMENTS = ['laurel==0.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_NAME): cv.string,
```

Comment:
Is this username as in credential? Then use `CONF_USERNAME` constant from const.py.